### DISCUSSION TITLE
Give the possibility to scan a barcode via CameraView

### DISCUSSION THREAD

**Discussion by acaliaro** [CommunityToolkit/Maui#1998] [New Feature Discussions]
Currently CameraView does not allow you to read a barcode. Via this article

https://vladislavantonyuk.github.io/articles/Various-methods-for-barcode-scanning-in-.NET-MAUI/

it is explained how to do this using a method external to CameraView.
Is it possible to try to integrate this method directly into CameraView?

**Comment by bijington** [comment]
Yes eventually it will. The approach will be based on the limited details mentioned in this discussion. https://github.com/CommunityToolkit/Maui/discussions/1687

I think we are postponing the community standup to next week. I will aim to write something up by then so we can potentially discuss it
